src/Disks/IDisk.h

```diff
@@ -367,7 +367,13 @@ class IDisk : public Space
     /// Actually it's a part of IDiskRemote implementation but we have so
     /// complex hierarchy of disks (with decorators), so we cannot even
     /// dynamic_cast some pointer to IDisk to pointer to IDiskRemote.
-    virtual MetadataStoragePtr getMetadataStorage() = 0;
+    virtual MetadataStoragePtr getMetadataStorage()
```

Comment:
Why we have implementation now?

Reply:
because no need to require implementation of this method from all disks, only from DiskObjectStorage, so it is ok to throw NOT_IMPLEMENTED here.